lib/src/protocol/parser.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:convert';
+
+import 'package:tbdex/src/protocol/models/close.dart';
+import 'package:tbdex/src/protocol/models/message.dart';
+import 'package:tbdex/src/protocol/models/offering.dart';
+import 'package:tbdex/src/protocol/models/order.dart';
+import 'package:tbdex/src/protocol/models/order_status.dart';
+import 'package:tbdex/src/protocol/models/quote.dart';
+import 'package:tbdex/src/protocol/models/resource.dart';
+import 'package:tbdex/src/protocol/models/rfq.dart';
+
+abstract class Parser {
+  static Message parseRawMessage(String rawMessage) {
```

Comment:
Mostly a question for me, but why not call these `parseMessage` or `parseResource`? Is there something else using that? Or is Raw more clear here?

Reply:
mainly because on tbdex-js, there's a `rawToMessageModel()` (which is called in `parseMessage()`) that was squished into one function here, but having raw in the method name doesn't make it more clear.. will remove it!